guides/Tong/README.md

```diff
@@ -0,0 +1,173 @@
+## Written by Sharon Tong
+_________________________
+
+### Contact
+- Slack: @Sharon on wcscusf.slack.com
+- [LinkedIn](linkedin.com/in/sharon-tong)
+- Email: sharont1@mail.usf.edu
+### Table of Contents
+1. [Prerequisites](#prereq)
+2. [Summary](#summary)
+3. [Installing PFsense](#installation)
+4. [Configuring PFsense](#configuration)
+5. [Troubleshooting](#troubleshooting)
+
+## Prerequisites <a id="prereq"></a>
+1. Have your [virtual environment](https://www.virtualbox.org/wiki/Downloads) configured.
```

Comment:
When this is merged in along with Nestor's and Jon's guides make another pull request with the new links. For now this is good, just a note for the future.